plugin/mlsdbonlinelocator.cpp

```diff
@@ -94,21 +104,21 @@ bool MlsdbOnlineLocator::findLocation(const QList<MlsdbProvider::CellPositioning
         return true;
     }
 
-    if (!haveFieldData(cells)) {
-        // no field data available
-        qCDebug(lcGeoclueMlsdbOnline) << "No field data available for MLS online request";
-        return false;
-    }
-
-    QNetworkRequest req(QUrl("https://location.services.mozilla.com/v1/geolocate?key=" + m_mlsKey));
-    req.setHeader(QNetworkRequest::ContentTypeHeader, "application/json");
-
     QVariantMap map;
-    map.unite(globalFields());
     map.unite(cellTowerFields(cells));
     map.unite(wifiAccessPointFields());
+    if (map.isEmpty() && !(m_fallbacksLacf || m_fallbacksIpf)
+    {
```

Comment:
Please do `) {` for consistency with the rest of the code, i.e. put opening bracket on the same line. It's not that one style is better then the other, just let's not mix them.